base/ntuple.jl

```diff
@@ -42,6 +42,20 @@ ntuple(f, ::Val{1}) = (@_inline_meta; (f(1),))
 ntuple(f, ::Val{2}) = (@_inline_meta; (f(1), f(2)))
 ntuple(f, ::Val{3}) = (@_inline_meta; (f(1), f(2), f(3)))
 
+"""
+    ntuple(f::Function, ::Val{N})
+
+Create a tuple of length `N`, computing each element as `f(i)`,
+where `i` is the index of the element. By taking a `Val{N}` 
+argument, is able to generate more efficient code than the
```

Comment:
I think there needs to be something before "is able". Perhaps change it to "it is possible to generate more..."

Reply:
>However, with constant propagation, I wonder if this is true in practice.

 Maybe "By passing `Val{N}()` instead of `N`, one can help type inference. This is necessary even for source-code literals beyond N=10; e.g. `ntuple(identity, 11)` will fail to infer. On the other hand, `ntuple(fun, N)` is preferable to `ntuple(fun, Val{N}())` in cases where `N` cannot be determined at compile time."

Reply:
Can we prefer `Val(N)` over `Val{N}()` ?